### PR TITLE
10M Seed Script

### PR DESCRIPTION
**Pull request by ucb15erickao**

### CODE REVIEW

File: database/10M.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const faker = require('faker');
```
Comment (ryanminjaeshin) [inline database/10M.js:1]: 👍

File: database/schema.cql [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  title text,
+  description text,
+  tag int,
+  rating int,
+  review_count int,
+  PRIMARY KEY (id)
+);
+
+
+CREATE KEYSPACE hotel WITH replication =
```
Comment (ryanminjaeshin) [inline database/schema.cql]: Is this keyspace intended?

**Review by ryanminjaeshin — APPROVED** [review]
👍

File: database/schema.cql (hunk 2 above)
Comment (ucb15erickao) [reply database/schema.cql]: oh no this is example from docs that I should delete

**Pull request merged**